docs/user_manual/hosting_a_game_server.md

```diff
@@ -62,16 +68,39 @@ connect to the replacement login server, follow these steps:
     
     **Do not manually open these ports in the firewall on the machine where the game server runs,
       otherwise votekick may not work correctly. taserver itself will manage the firewall rules**
+   
+    In the router, you will need to redirect those ports on the machine you are hosting the server on.
```

Comment:
-> "**to** the machine you are hosting the server on"

Reply:
fixed